app/views/api/v3/analytics/user_analytics/show.html.erb

```diff
@@ -110,28 +108,35 @@
 
   <!-- monthly hypertension control view -->
   <div class="card">
-    <h3>Hypertension control</h3>
+    <h3>Patients with hypertension controlled</h3>
     <p class="desc">
-      How many patients with hypertension had BP below 140/90 at their last follow-up visit in the month?
+      Hypertension patients with BP &lt;140/90 at their most recent visit in the last 3 months
     </p>
 
-    <table class="progress-table">
-      <% @user_analytics.monthly_period_list.each do |month_date| %>
-        <tr>
-          <td class="row-label no-padding"><%= month_date.strftime("%B") %></td>
-          <td>
-            <span class="percent">
-              <%= @user_analytics.monthly_htn_control_rate(month_date) %>
-            </span>
-          </td>
-          <td>
-            <%= @user_analytics.monthly_htn_stats_by_date(:controlled_visits, month_date) %>
-            of
-            <%= @user_analytics.monthly_htn_stats_by_date(:follow_ups, month_date) %>
-          </td>
-        </tr>
-      <% end %>
+    <p class="desc control-desc">
+        <strong><%= @user_analytics.monthly_htn_control_last_period %></strong>
+        <br><%= @user_analytics.monthly_htn_control_last_period_patient_counts %> patients
+        <br><strong class="green"><%= @user_analytics.monthly_htn_control_last_control_rate %>%</strong>
+    </p>
+
+    <table class="bar-chart">
+        <tbody>
+            <tr>
+                <% @user_analytics.htn_control_monthly_period_list.each do |month_date| %>
+                  <% control_rate = @user_analytics.monthly_htn_control_rate(month_date) %>
+                  <td class="bar-row">
+                    <div class="bar-value"><span><%= control_rate %>%</span></div>
+                    <div style="height: <%= control_rate * 2 %>px;" class="bar"><span></span></div>
+                  </td>
+                <% end %>
```

Comment:
What does this look like now? 👀

Reply:
Good call, let me add some screenshots to the PR.